frontend/src/queries/nodes/InsightViz/InsightViz.tsx

```diff
@@ -51,13 +51,14 @@ export function InsightViz({ uniqueKey, query, setQuery, context, readOnly }: In
     }
 
     const showIfFull = !!query.full
-    const disableHeader = query.showHeader ? !query.showHeader : !showIfFull
-    const disableTable = query.showTable ? !query.showTable : !showIfFull
-    const disableCorrelationTable = query.showCorrelationTable ? !query.showCorrelationTable : !showIfFull
-    const disableLastComputation = query.showLastComputation ? !query.showLastComputation : !showIfFull
-    const disableLastComputationRefresh = query.showLastComputationRefresh
-        ? !query.showLastComputationRefresh
-        : !showIfFull
+    const disableHeader = !(query.showHeader ?? showIfFull)
```

Comment:
tempting to suggest flipping all of these so they become `showHeader` , `showTable` etc so you don't have to mentally parse "the header won't be shown if show header is not true or if when it is null then......"

but that's scope creep here 😊

Reply:
Yeah, I tend to agree. I'm unsure why it was the way it is but thought it safer to make minimal changes given it's something the Analytics team are working on. Happy to put a PR together

Reply:
Well, don't dig too deep... it's a side quest if anything and you might find my dirty fingers at the base of it anyway 🤣